web/src/components/domains/task/TaskItem/TaskItem.jsx

```diff
@@ -1,11 +1,68 @@
 import styles from './TaskItem.module.css';
+import PropTypes from 'prop-types';
 
 /*
 Please create the <TaskItem /> component following the design from the Figma file.
 Please make sure to add styles using CSS Modules.
 Add the necessary props to the component.
 */
 
-export function TaskItem(props) {
-  return <div className={styles.itemWrapper}></div>;
+function TaskItem(props) {
+  function colorBox() {
+    if (props.priority === 'low') {
+      return styles.btnGreen;
+    }
+    if (props.priority === 'medium') {
+      return styles.btnYellow;
+    }
+    if (props.priority === 'high') {
+      return styles.btnRed;
+    }
+  }
+  return (
+    <div className={styles.mainContainer}>
+      <h4>{props.ProjectName}</h4>
+      <button className={colorBox()}>{props.priority}</button>
```

Comment:
I think here using a div would be the better choice. 

```suggestion
      <div className={colorBox()}>{props.priority}</div>
```